pkg/sharings/sharings.go

```diff
@@ -204,8 +202,39 @@ func ShareDoc(instance *instance.Instance, sharing *Sharing, recStatus *Recipien
 			}
 		}
 
+		var values []string
+
+		// Dynamic sharing
+		if rule.Selector != "" {
+			// Create index based on selector to retrieve documents to share
+			index := mango.IndexOnFields(docType, sharing.ID(), []string{rule.Selector})
+			err := couchdb.DefineIndex(instance, index)
+			if err != nil {
+				return err
+			}
+
+			var docs []couchdb.JSONDoc
+
+			// Request the index for all values
+			for _, val := range rule.Values {
```

Comment:
I tried that, but I receive a `CouchDB(no_usable_index): There is no index available for this selector.`.
I pass a `Filter` slice, as a parameter : `mango.Or(filters...)`, I don't know if I do something wrong.